subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/CachedScalaCompileIntegrationTest.groovy

```diff
@@ -39,11 +39,11 @@ class CachedScalaCompileIntegrationTest extends AbstractCachedCompileIntegration
             }
 
             dependencies {
-                compile group: 'org.scala-lang', name: 'scala-library', version: '2.11.8'
+                compile group: 'org.scala-lang', name: 'scala-library', version: '2.11.11'
```

Comment:
:o: There are several places where we refer to Scala versions matching `/2\.11\.\d+/` that have not yet been upgraded. Should we do so?